README.md

```diff
@@ -13,6 +13,40 @@ composer require prestashop/composer-script-handler --dev
 
 In your Shop, you can now declare modules into "prestashop.modules" in the "extras" section of Composer.
 
-Once you do that, on "install" and "update" processes, every module will be installed so you will always get the most "up to date" version of each module!
+
+
+Once you do that, on "install" process, the list of modules will be installed and can be overwritten on demand.
+
+### Configuration
+
+You can configure the list of modules, the number of process to paralleling the download of modules and the timeout to wait for download status.
+
+* modules: this is the list of Composer packages with a version, they **must** be of "prestashop-module" type;
+* processes: it's the number of parallel process allowed to download modules (**8** by default)
+* frequency: the time to wait before check the status of each current processes,in ms (**200** by default)
+
+```js
+{
+    "name": "my/shop",
+    "...": "...",
+    "scripts": {
+        "post-install-cmd": [
+            "PrestaShop\\Composer\\ScriptHandler::install"
+        ]
+    },
+    "extra": {
+        "prestashop": {
+            "modules": {
+                "prestashop/blockreassurance": "^3",
+                "prestashop/contactform": "^4",
+                "prestashop/dashactivity": "^2",
+                "prestashop/dashgoals": "^2"
+            },
+            "processes": 12,
+            "frequency": 80
```

Comment:
I'm juste having a little doubt about this one, I think `frequency` alone doesn't reflect why this parameter is used Maybe we should use `processUpdateFrequency` or `processCheckingFrequency`
I know it's longer but it's also clearer.

Reply:
hmm, how about: 

```js
"prestashop" : {
    "modules": { ... },
    "process-manager": {
        "parallel-processes": 12,
        "frequency-update": 100
    }
}
```